LICENSE

```diff
@@ -2,6 +2,7 @@ Copyright 2021 European Spallation Source
 Modifications Copyright 2021 UK Research and Innovation
 All rights reserved
 
+
```

Comment:
Added to fix caching issue in GitHub actions (a broken cache upload meant that the cache couldn't be overwritten or read from). Needed a new commit SHA to generate a new cache key.